cli/src/tests/parser_test.rs

```diff
@@ -502,6 +502,80 @@ h + i
     );
 }
 
+#[test]
+fn test_parsing_after_editing_tree_that_depends_on_column_position() {
+    let dir = fixtures_dir()
+        .join("test_grammars")
+        .join("depends_on_column");
+    let grammar = fs::read_to_string(dir.join("grammar.json")).unwrap();
+    let (grammar_name, parser_code) = generate_parser_for_grammar(&grammar).unwrap();
+
+    let mut parser = Parser::new();
+    parser
+        .set_language(&get_test_language(&grammar_name, &parser_code, Some(&dir)))
+        .unwrap();
+
+    let mut code = b"
+ x
+    "
+    .to_vec();
```

Comment:
What do you think about writing this declaration, and the assertions about `code` using single line string literals? I think it would be easier to read. And I don't think the trailing whitespace is needed (correct me if I'm wrong there).

I know that you insert a newline in the final step. I kinda think it would be fine to write the final code assertion with a newline escape sequence, like:

```rs
assert_eq!(code, "\n  x");
```

Reply:
> What do you think about writing this declaration, and the assertions about `code` using single line string literals? I think it would be easier to read. And I don't think the trailing whitespace is needed (correct me if I'm wrong there).
> 
> I know that you insert a newline in the final step. I kinda think it would be fine to write the final code assertion with a newline escape sequence, like:
> 
> ```rust
> assert_eq!(code, "\n  x");
> ```

ok all assertions are now single-line